components/datadog/datadog.app.mjs

```diff
@@ -0,0 +1,213 @@
+import { v1 } from "@datadog/datadog-api-client";
+import { v4 as uuid } from "uuid";
+import constants from "./actions/common/constants.mjs";
+
+export default {
+  type: "app",
+  app: "datadog",
+  propDefinitions: {
+    host: {
+      type: "string",
+      label: "Host",
+      description: "The name of the host that produced the metric",
+      async options({ prevContext }) {
+        const {
+          hostList,
+          totalReturned,
+        } = await this.listHosts({
+          start: prevContext.totalReturned ?? 0,
```

Comment:
Does the `totalReturned` from `listHosts()` include the hosts before `start`?

E.g., if `start` is `100`, and `(await this.listHosts({ start })).hostList.length === 20`, is `totalReturned` `20 `or `120`.